### PR TITLE
Exec tachikoma update 20150927163346

### PR DESCRIPTION
**Pull request by togusafish**
:hamster::hamster::hamster: <br /> <br /> Strategy none
Powered by <a href="http://tachikoma.io/?utm_source=github&utm_medium=pull_request&utm_campaign=build6053939230386075629">Interval Pull Request as a Service - Tachikoma.io</a> <br />Announcement: <a href="http://tachikoma.io/2015-06-28-updated-permissions.html?utm_source=github&utm_medium=pull_request&utm_campaign=build6053939230386075629">2015-06-28 Updated permissions - Tachikoma.io</a>

### CODE REVIEW

**Comment by j0k3r** [comment]
@sanemat are you aware of all these empty commit PR ?

**Comment by sanemat** [comment]
@j0k3r Yes. Interval pull request (with empty commit) kicks tests on travis-ci. Sometimes I see failing test. E.g. https://github.com/sanemat/tachikoma/pull/168 This is like a `health check`.

Can I help you?

`tachikoma` gem supports `none`, `bundler`, `carton`, `david`, `composer` and `cocoapods`.
But `tachikoma.io` web service supports `none`, `bundler`, `carton` and `david`.
http://tachikoma.io/configuration.html
Sorry for the inconvenience.

I recommend this article(in Japanese) for `composer` users and `cocoapods` users.
http://packsaddle.org/articles/interval-pull-request-update-overview/

TL;DR
* Kick interval dependency update (e.g. from heroku)
* Run `composer update` on your CI
* Send pull request

:smile_cat: Thanks
